lib/utmp.c

```diff
@@ -99,13 +99,26 @@ static void failtmp (const char *username, const struct utmp *failent)
 	 * Append the new failure record and close the log file.
 	 */
 
-	if (   (write_full(fd, failent, sizeof *failent) == -1)
-	    || (close (fd) != 0)) {
-		SYSLOG ((LOG_WARN,
-		         "Can't append failure of user %s to %s: %m",
-		         username, ftmp));
+	if (   (write_full(fd, failent, sizeof *failent) == -1)) {
```

Comment:
You could remove that whitespace now.

Reply:
Dropped spaces and duplicate pair of parenthesis.